stage_ros/CamImage image_pprev
stage_ros/CamImage image_prev
stage_ros/CamImage image_now
